js/main.js

```diff
@@ -1,23 +1,24 @@
-var user="John Doe";
+let user = "John Doe";
 console.log(user);
-var student="Olga";
+
+let student = "Olga";
 console.log(student);
-var user="student";
+
+user = "student";
```

Comment:
it is the same. if you dont write let,const,var directly - variable initialise as var